packages/babel-plugin-ember-test-metadata/__tests__/get-normalized-file-path-test.js

```diff
@@ -158,4 +159,41 @@ describe('getNormalizedFilePath', () => {
       expect(normalizedFilePath).toEqual(expectedPath);
     });
   });
+
+  describe('custom normalized filePath function', () => {
+    const appRoot = '/Users/tester/workspace/personal/test-bed/example-app';
+
+    function normalizeBuildIsolationPath(filePath, packageName, projectRoot) {
+        const pathSegments = filePath.split(sep);
+        const testFilePath = pathSegments
+          .slice(pathSegments.indexOf('tests'))
+          .join(sep);
+      
+        if (!projectRoot) {
+          return testFilePath;
+        }
+
+        const addonRoot = pathSegments.slice(0, pathSegments.lastIndexOf(packageName));
+        const projectBaseDir = basename(resolve(addonRoot.join(sep), projectRoot));
+        const projectPath = addonRoot.slice(addonRoot.indexOf(projectBaseDir) + 1).join(sep);
+        
+        return join(projectPath, testFilePath);
```

Comment:
I don't think we need all this superfluous stuff for the specific use case we're trying to support. Meaning, this can be simplified.

Reply:
Should be something along the lines of:
```js
function normalizeBuildIsolationPath({ filePath, packageName }) {
  const pathSegments = filePath.split(sep);
  const relativePathSegments = pathSegments.slice(pathSegments.indexOf(packageName) + 1);
  const normalizedPathSegments = relativePathSegments.filter((segment) => segment !== packageName);

  return normalizedPathSegments.join(sep);
}
```